pkg/convertor.go

```diff
@@ -78,6 +93,81 @@ func computeResourceID(resource parser.TerraformResource) string {
+	case "google_kms_crypto_key_iam_binding":
+		return fmt.Sprintf("%s %s", v("crypto_key_id"), v("role"))
+	case "google_kms_crypto_key_iam_member":
+		return fmt.Sprintf("%s %s %s", v("crypto_key_id"), v("role"), v("member"))
+	case "google_organization_iam_member":
+		return fmt.Sprintf("%s %s %s", v("org_id"), v("role"), v("member"))
+	case "google_project_iam_member":
+		conditions, ok := resource.AttributeValues["condition"].([]any)
+		if ok && len(conditions) >= 1 {
+			condition = conditions[0].(map[string]any)
+			return fmt.Sprintf("%s %s %s %s", v("project"), v("role"), v("member"), condition["title"])
+		}
+		return fmt.Sprintf("%s %s %s", v("project"), v("role"), v("member"))
+	case "google_project_iam_binding":
+		return fmt.Sprintf("%s %s", v("project"), v("role"))
+	case "google_project_iam_custom_role":
+		return fmt.Sprintf("%s %s", v("project"), v("id"))
+	case "google_sql_database_instance":
+		return fmt.Sprintf("projects/%s/instances/%s", v("project"), v("name"))
+	case "google_sql_user":
+		return fmt.Sprintf("%s/%s/%s", v("project"), v("instance"), v("name"))
+	case "google_iap_tunnel_instance_iam_binding":
+		return fmt.Sprintf("%s %s", v("instance"), v("role"))
+	case "google_secret_manager_secret_iam_binding":
+		return fmt.Sprintf("%s %s", v("secret_id"), v("role"))
+	case "google_secret_manager_secret_iam_member":
+		return fmt.Sprintf("%s %s %s", v("secret_id"), v("role"), v("member"))
+	case "google_storage_bucket_iam_member":
+		return fmt.Sprintf("%s %s %s", v("bucket"), v("role"), v("member"))
+	case "google_storage_bucket_iam_binding":
+		return fmt.Sprintf("%s %s", v("bucket"), v("role"))
+	case "google_compute_subnetwork_iam_binding":
+		return fmt.Sprintf("%s %s", v("subnetwork"), v("role"))
+	case "google_pubsub_topic_iam_binding":
+		return fmt.Sprintf("%s %s", v("topic"), v("role"))
+	case "google_pubsub_topic_iam_member":
+		conditions, ok := resource.AttributeValues["condition"].([]any)
+		if ok && len(conditions) >= 1 {
+			condition = conditions[0].(map[string]any)
+			return fmt.Sprintf("%s %s %s %s", v("topic"), v("role"), v("member"), condition["title"])
+		}
+		return fmt.Sprintf("%s %s %s", v("topic"), v("role"), v("member"))
+	case "google_pubsub_subscription_iam_member":
+		return fmt.Sprintf("projects/%s/subscriptions/%s %s %s", v("project"), v("subscription"), v("role"), v("member"))
+	case "google_pubsub_subscription_iam_binding":
+		return fmt.Sprintf("%s projects/%s/subscriptions/%s %s", v("project"), v("project"), v("subscription"), v("role"))
+	case "google_resource_manager_lien":
+		return fmt.Sprintf("%s/%s", strings.ReplaceAll(v("parent"), "projects/", ""), v("name"))
+	case "google_monitoring_uptime_check_config":
+		return fmt.Sprintf("%s %s", v("project"), v("id"))
+	case "google_monitoring_alert_policy":
+		return fmt.Sprintf("%s %s", v("project"), v("name"))
+	case "google_monitoring_notification_channel":
+		return fmt.Sprint(v("name"))
```

Comment:
i think the `fmt.Sprint` can be dropped here and the statement can become `return v("name")`